src/zope/app/apidoc/codemodule/browser/menu.pt

```diff
@@ -22,13 +22,13 @@
 
     <div tal:define="classes view/findClasses"
          tal:condition="classes">
-
-      <a href="" target="main"
-         tal:repeat="info classes"
-         tal:attributes="href info/url"
-         tal:content="info/path">
+      <div tal:repeat="info classes" style="display:block">
```

Comment:
Is there some CSS that changes the presentation of divs that requires an explicit style="display: block" here?

Reply:
There is no such style rule, and I think using line breaks between links makes much more sense anyway.

Reply:
Huh?  `<div>` elements are block-level elements by default, so you should be able to drop the `style` attribute and get the desired effect anyway?  I feel like I'm misunderstanding something here.